Homework-1 pull request

vidar22 opened the pull request.

vidar22 committed:
first

vidar22 commented on web/css/style.css:
```diff
@@ -1,12 +1,16 @@
 }
 h1{
 	font-size: 40px;
 	line-height: 28px;
 	color: #fffefe;
-	font-family: museo Moderno;
+	font-family: MuseoSlab;
```
Nice work!

vidar22 committed:
Test Feature

vidar22 committed:
services - done

edit index; css.

vidar22 committed:
features done

I am found mistake about h2 h3, but I won't fix it. because you have to rewrite everything css files.
I want finished all page))))

FlashMX2004 commented on web/css/services.css:
```diff
@@ -0,0 +1,64 @@
+.services{
+	margin-top: 100px;
+
+}
```
Не делай секциям section отступы. А вдруг нужно будет фон поставить? С отступом будет пустое пространство.
Отступы сделай для внутренних элементов.

FlashMX2004 commented on web/css/services.css:
```diff
@@ -0,0 +1,64 @@
+	justify-content: space-around;
+	margin-left: 100px;
+	margin-top: 125px;
+	margin-right: 100px;
```
Моя рекомендация для всех тегов `<section>` обернуть их содержимое в какой-нибудь div с классом section-inner например, а section-inner прописать в глобальном файле стилей style.css. Ибо зачем прописывать каждому блоку эти отступы? _Если посмотреть на макете, можно заметить, что у всех секций, их содержимое имеет одинаковые боковые отступы._
Что я имею ввиду:
```
<!-- Head Section -->
<section>
    <div class="section-inner">
        ...
    </div>
</section>

<!-- Services Section -->
<section>
    <div class="section-inner">
        ...
    </div>
</section>

<!-- Features Section -->
<section>
    <div class="section-inner">
        ...
    </div>
</section>
```
В style.css:
```
...
.section-inner {
    margin-left: 100px;
    margin-right: 100px;
    /* Или в одну строку*/
    margin: 0 100px;
}
...
```

FlashMX2004 commented on web/index.html:
```diff
@@ -3,129 +3,170 @@
+							<a class="nonecolor" href="#">PORTFOLIO</a>
+							<a class="nonecolor" href="#">CONTACT US</a>
+							<a class="nonecolor" href="#">BLOG</a>
 						</span>	
```
Вместо класса nonecolor можно сделать стиль для:
```
.menu a { ... }
```

FlashMX2004 commented on web/index.html:
```diff
@@ -3,129 +3,170 @@
 				</div>
-				<div>
+				<div class="dividor">
+					<img src="img/dividorhorizontal.png" alt="img">
```
Не, ну это можно было через border :)

vidar22 committed:
otherservices done.

I am small fixed services;style; and img. Created "otherservices.css"
flaby.psd?
-I don't touch Flaby.psd!!   =))
see you later.

vidar22 committed:
fixed fonts..

i'm sorry small tired, fogot about fonts.
waked up six o'clock

FlashMX2004 commented on web/css/otherservices.css:
```diff
@@ -0,0 +1,99 @@
+}
+.otherservicescontent{
+	margin-left: 100px;
+	margin-right: 100px;
```
Опять же таки, этих отступов можно было бы избежать, если бы ты делал через section-inner

FlashMX2004 commented on web/css/otherservices.css:
```diff
@@ -0,0 +1,99 @@
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/gamedesign.png') left no-repeat;	
+
+}
+.responsivedesign{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/responsivedesign.png') left no-repeat;	
+
+}
+.freeshopping{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/freeshopping.png') left no-repeat;	
+
+}
+.bonusdownload{
+		width: 330px;
+	height: 50px;
+	padding: 10px;
+	padding-top: 5px;
+	padding-left: 0px;
+	margin: 0px;
+	list-style-type: none;
+	background: url('../img/3otherServices/bonusdownload.png') left no-repeat;	
+
+}
```
ТАК Я НЕ ПОНЯЛ?!! Это что ещё закопипаста??? Выдерни это к чертовой матери!! Я бы вообще стёр это всё, и через имг сделал бы, но раз тебе нравится через css, то это вообще в пару строк можно.
Пиши:
```suggestion
.otherservicescontent ul {
	width: 330px;
	height: 50px;
	padding: 10px;
	padding-top: 5px;
	padding-left: 0px;
	margin: 0px;
	list-style-type: none;
}

.developing{ background: url('../img/3otherServices/ideadeveloping.png') left no-repeat; }
.coding{ background: url('../img/3otherServices/bestcoding.png') left no-repeat; }
.gamedesign{ background: url('../img/3otherServices/gamedesign.png') left no-repeat; }
.responsivedesign{ background: url('../img/3otherServices/responsivedesign.png') left no-repeat; }
.freeshopping { background: url('../img/3otherServices/freeshopping.png') left no-repeat; }
.bonusdownload { background: url('../img/3otherServices/bonusdownload.png') left no-repeat; }
```

FlashMX2004 commented on web/index.html:
```diff
@@ -3,169 +3,214 @@
+							<img src="img/0header/Button login.png" alt="login">
+							</a>	
+					</div>
+					<div class="underline"/>	
```
Fix
```suggestion
					<div class="underline">	
```

FlashMX2004 commented on web/index.html:
```diff
@@ -3,169 +3,214 @@
+					<div>					
+						<div class="content-flex">	
+							<ul class="developing">
+								<li">
```
Fix
```suggestion
								<li>
```

vidar22 committed:
work on mistakes

thank you very much.
you are realy help me =)

FlashMX2004 commented on web/css/style.css:
```diff
@@ -29,190 +37,55 @@ h4{
+	width: 28px;
+	height: 1px;
+	color: #2eafeb;
+	border: 1px solid;
```
Лучше просто `border` заменить на `border-top` или `border-bottom`, тк просто `border` делает тебе 4 границы (`border-top`, `border-right`, `border-bottom`, `border-left`), и наделяет их шириной в 1px, соответственно граница в высоту получается 2px, тк задействованы `border-top` и `border-bottom` одновременно

vidar22 replied:
да, я думал над этим, поэтому пришел к значениям 28х1  , так как результат должен быть 30х3 рамка добавляет по 1п с какаждой стороны и получается заданный размер с макета.
*Но смысл я понял, в идеале, можно обойтись без рамки вообще?*

FlashMX2004 commented on web/css/style.css:
```diff
@@ -29,190 +37,55 @@ h4{
-	width: 168px;
-	height: 51px;
-	margin: 45px 0px 36px 245px;
+.innner-content{
+	margin-left: 100px;
+	margin-right: 100px;
```
Раз сделал, то будь добр, оберни ВСЕ внутренние содержимые секций :D

FlashMX2004 commented on web/css/otherservices.css:
```diff
@@ -0,0 +1,56 @@
+
+}
+.otherservicescontent{
+	margin-left: 100px;
+	margin-right: 100px;
```
Убрать, когда контент секции otherservices будет обернут в `inner-content`
```suggestion
```

vidar22 replied:
.innner-content
злосчастный иннер. я тут запутался уже дважды. Я делаю себе закладку про это, так не оставлю.

FlashMX2004 commented on web/index.html:
```diff
@@ -3,169 +3,209 @@
 			<div>
-				<h3 class="section-UC-h3">We have great features with</h3>
-
-				<h2 class="section-UC-h2">Unlimited Customization Features</h2>
+				<img class="phones" src="img/2UnlimitedCustomizationFeatures/phone.png" alt="phone">
+			</div>	
+			<div class="featurescontent">
```
Тг-аа как грязно! Во-первых, этот див с картинкой сдвигает контент, вместо того чтобы быть действительно бэкграундом и не участвовать в разметке (position: absolute).
Во-вторых, кроме того, что нужно будет обернуть в `inner-content` как говорилось, так я ещё посмотрел на макет, нужно будет этот `inner-content` разделить надвое, и контент засунуть в правую половину. Справишься?

Если есть вопросы, можешь тут написать

vidar22 replied:
оуоу тормози, я видимо не туда display: none поставил.
конечно это порнуха
А эти классы в разметке с начальной верстки class="section-UC-h3"
Буду все блочить чтоб не вылезало то, что я еще не *переписывал*.
У меня получается такая картина : я написал разметку; дал ей css правила не понимая как называть их правильно и что писать в них.
Затем ты начал меня учить *как тут все живет на самом деле* и *забудь все что ты знаешь* поэтому я теперь на основе старой разметки переписываю.
оч много букв... ХD

inner content меня преследует..

vidar22 replied:
!!!!!!!!!!!!!!!

FlashMX2004 replied:
--------

FlashMX2004 replied:
Test

FlashMX2004 reviewed (commented):
Уф, недоглядел конечно с features секцией :)

vidar22 committed:
reviews done

added css reviews
fix style.css
fix name img
flaby.psd????? ><
I dont touch flaby!!
=)

vidar22 commented:
Test?

vidar22 committed:
freeresourse done

big work was today
fixed all blocks and add new block
i'm happy)))
thanks!

vidar22 committed:
price done

good work)
this is drive

vidar22 committed:
subscribe done

ok

FlashMX2004 commented on web/index.html:
```diff
@@ -3,246 +3,360 @@
+		</section>
+		<!-- Subscribe Section -->
+		<section class="subscribe">
+			<div class="subscribe-main">
```
Нету `inner-content` класса

FlashMX2004 commented on web/index.html:
```diff
@@ -3,246 +3,360 @@
-				</a>
-			</div>
-		</section>
+			<div class="dividorhorizontal"></div>
```
`price-title` и `dividorhorizontal` не обернуты в `inner-content`

FlashMX2004 replied:
Это в  Prices Section

FlashMX2004 replied:
А так в целом, довольно быстро разошелся ))

vidar22 replied:
good! 
work about mistakes, i will do next week =P 
=P
today is game over.

vidar22 committed:
footer done

freelance..
I am go to you.

good day! =)

vidar22 commented:
good!
work about mistakes, i will do next week =P
=P
today is game over.

vidar22 commented:
Как
Как сделать так чтобы в header телефон при маштабировании не убегал.
Как сделать так чтобы в price plan числа ценников были такими же жирными и толстыми как в макете.
Как надо настроить кнопку subscribe?

в Prices Section inner добавлен.
footer отрегулированы отступы.

vidar22 committed:
fix footer

I can questions and send you them  in pull requests))

FlashMX2004 commented:
**Как сделать так чтобы в header телефон при маштабировании не убегал**
Во первых, сделай первой секции overflow: hiddeb;
Можешь еще сделать телефон вне `section-inner` и дать ему position: absolute; bottom: 0;

**Как сделать так чтобы в price plan числа ценников были такими же жирными и толстыми как в макете**
А разве они не такие же как на макете?)

**Как надо настроить кнопку subscribe?**
В каком плане настроить, если чтобы что-то происходило по клику, то это js

vidar22 committed:
september 2022

vidar22 committed:
edit link main page

and create {hover/visited} for button login
